src/dcd/server/autocomplete/util.d

```diff
@@ -758,10 +758,59 @@ AutocompleteResponse.Completion makeSymbolCompletionInfo(const DSymbol* symbol,
 	string definition;
 	if ((kind == CompletionKind.variableName || kind == CompletionKind.memberVariableName) && symbol.type)
 		definition = symbol.type.name ~ ' ' ~ symbol.name;
-	else if (kind == CompletionKind.enumMember)
-		definition = symbol.name; // TODO: add enum value to definition string
 	else
-		definition = symbol.callTip;
+	{
+		switch (kind)
+		{
+		case 'e':
```

Comment:
you should probably use the CompletionKind enum here and perform a final switch on it.

Reply:
Done, using the enum now, and can't use final, because we rely on the default to fallback to the calltip, we don't need everything